test/functional/test_site/testGoogleAnalytics.md

```diff
@@ -0,0 +1,18 @@
+# Test Google Analytics Plugin
+
+Following script should be included:
+
+```js
+<!-- Global site tag (gtag.js) - Google Analytics -->
+<script async src="https://www.googletagmanager.com/gtag/js?id=TRACKING-ID"></script>
+<script>
+  window.dataLayer = window.dataLayer || [];
+
+  function gtag() {
+    dataLayer.push(arguments);
+  }
+  gtag('js', new Date());
+
+  gtag('config', 'TRACKING-ID');
```

Comment:
I don't think it is necessary to put the content of the script here (and as a matter of fact, this page won't be needed at all). Although it is convenient for reviewers, it also means that future updates to the main script would require additional work in replacing it here as well. New changes would immediately surface on the diff of the expected files anyway, making them quite obvious, so this won't be necessary.